services/server/cors.ts

```diff
@@ -0,0 +1,11 @@
+export const setAllowOriginIfTrusted = (responseHeaders, origin) => {
+  if (trustedOrigins.includes(origin)) {
+    responseHeaders.headers["Access-Control-Allow-Origin"] = origin;
+  }
+};
+
+const trustedOrigins = [
```

Comment:
Move the configuration to `.env` so it can be later configured by the pipeline and ssw.com.au.

Reply:
We can leave this as is since it's probably going to be used by the same origin for the foreseeable future, and we just need this to support staging/production Blob Storage URLs.